render.go

```diff
@@ -71,6 +72,21 @@ func renderFailedSpecContext(space, text string, failure types.SpecFailure) {
 	renderNewLine()
 }
 
+func renderPanickedSpecContext(space, text string, failure types.SpecFailure) {
+	renderLine(space, rbf(Icons.panicked), rf(text))
+	renderNewLine()
+	scanner := bufio.NewScanner(strings.NewReader(failure.Location.FullStackTrace))
+	for scanner.Scan() {
+		renderText(space, strings.Replace(scanner.Text(), "\t", "    ", 1))
```

Comment:
Maybe we could use 2 spaces, what do you think?

Reply:
Not a problem. I am so used to 4 spaces that I hadn't noticed the 2 spaces in the specs texts.